packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -66,3 +75,30 @@
     @mixin mobile-footer-xs-layout;
   }
 }
+
+[part='generated-answer-footer'] {
+  &.show-button-hidden {
+    @apply justify-end;
+    [part='answer-show-button'] {
+      @apply hidden;
+    }
+  }
+
+  @container (max-width: 600px) {
+    @mixin mobile-generated-answer-footer-layout;
+  }
+
+  @screen mobile-only {
+    @mixin mobile-generated-answer-footer-layout;
+  }
+}
+
+[part='generated-container'] {
+  &.answer-collapsed {
+    @apply max-h-[250px] overflow-hidden relative content-[''];
+  }
+  &.answer-collapsed:before {
+    @apply h-full w-full absolute top-0 left-0 content-[''];
+    background: linear-gradient(transparent 180px, white);
```

Comment:
Considering the collapsible height can be modified, this gradient has to update accordingly.